packages/dashmate/src/listr/prompts/validators/validatePort.js

```diff
@@ -0,0 +1,9 @@
+function validatePort(port) {
+  const portNumber = Math.floor(Number(port));
+
+  return portNumber >= 1
```

Comment:
Are there any reserved port we would like to avoid using?

Reply:
I don't have any in my mind. @strophy ?

Reply:
We should probably avoid using ports in the 1-1023 range, since superuser privileges are required to bind to these well-known ports under many systems. Other than that, ports > 1024 are fair game I think.

Reply:
We gonna use 443 for platform HTTP port on mainnet :)